python/dask_cudf/dask_cudf/sorting.py

```diff
@@ -218,9 +218,10 @@ def quantile_divisions(df, by, npartitions):
                 divisions[col].iloc[-1] += 1
                 divisions[col] = divisions[col].astype(dtype)
             else:
-                divisions[col].iloc[-1] = chr(
-                    ord(divisions[col].iloc[-1][0]) + 1
-                )
+                if last := divisions[col].iloc[-1]:
+                    divisions[col].iloc[-1] = chr(ord(last[0]) + 1)
+                else:
+                    divisions[col].iloc[-1] = chr(1)  # b/c "" < chr(1)
```

Comment:
Interestingly, using `chr(0)` here doesn't work, because `drop_duplicates` below drops it by treating `""` and `chr(0)` the same, even though `"" < chr(0)`. Do we care that `chr(1)` is not printable?

Reply:
#12990.

Reply:
> it suffices to provide any non-empty string

Yup. How about `"wence was here"`?

Reply:
I like David's suggestion (or we can wait for #12991 and use `chr(0)`)